internal/assets/opencode_review_transport_plugin_test.go

```
@@ -246,6 +262,242 @@ printf '%s\n' "$complete" >> "$GENTLE_AI_RELAY_LOG"
+	command.Stderr = stdoutBuf
+	if err := command.Start(); err != nil {
+		t.Fatalf("start barrier harness: %v", err)
+	}
+	for time.Now().Before(time.Now().Add(15 * time.Second)) {
+		if _, err := os.Stat(barrierPath); err == nil {
+			break
+		}
+		time.Sleep(50 * time.Millisecond)
+	}
+	if _, err := os.Stat(barrierPath); err != nil {
+		command.Process.Kill()
```

golangci-lint (2.12.2)

[error] Error return value of `command.Process.Kill` is not checked

(errcheck)